packages/react-router/modules/StaticRouter.js

```diff
@@ -90,7 +90,7 @@ class StaticRouter extends React.Component {
 
   handleBlock = () => noop;
 
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
StaticRouter is mainly for server rendering, so this lifecycle method won't fire on the server. It should be in a constructor instead.